### PR TITLE
Upgrade to MUI 5

### PR DESCRIPTION
**Pull request by seavor**
Update Material UI v4 to v5.

### CODE REVIEW

**Comment by ebbit1q** [comment]
does this also include and thus supersede #4599 ?

**Comment by seavor** [comment]
> 

Looks like it :)

**Review by ZeldaZach — APPROVED** [review]

File: webclient/src/types/server.tsx [hunk 1]
```diff
@@ -54,6 +54,13 @@ export class Host {
 }
 
 export const DefaultHosts: Host[] = [
+  {
+    name: 'Chickatrice',
+    host: 'mtg.chickatrice.net',
+    port: '443',
+    localPort: '4748',
```
Comment (brentmclark) [inline webclient/src/types/server.tsx:61]: Is this something that should be documented in the README file?

**Review by brentmclark — APPROVED** [review]

File: webclient/package.json [hunk 2]
```diff
@@ -3,10 +3,11 @@
   "version": "1.0.0",
   "private": true,
   "dependencies": {
-    "@material-ui/core": "^4.12.3",
-    "@material-ui/icons": "^4.11.2",
-    "@material-ui/lab": "^4.0.0-alpha.60",
-    "@material-ui/styles": "^4.11.4",
+    "@emotion/react": "^11.8.2",
+    "@emotion/styled": "^11.8.1",
```
[the remark this answers is not in the document]
Comment (seavor) [reply webclient/package.json:7]: I believe it's a dependency.

File: webclient/src/types/server.tsx (hunk 1 above)
Comment (seavor) [reply webclient/src/types/server.tsx:61]: Possibly, the README is missing a lot though.

File: webclient/package.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (brentmclark) [reply webclient/package.json:7]: Ah, ok.  Thanks.

**Pull request merged**